client/lib/ffi.rs

```diff
@@ -551,6 +551,8 @@ pub extern "C" fn casper_get_state_root_hash(
 ///
 /// See [get_item](super::get_item) for more details.
 #[no_mangle]
+#[allow(deprecated)]
```

Comment:
We shouldn't `allow` here - only at any callsites.

Reply:
Right, so it would be directly at the callsite not at the function declaration??

Reply:
Moved to callsite